.travis.yml

```diff
@@ -5,11 +5,12 @@ node_js:
 before_install:
   - npm install -g npm@2.13.5
   - npm install -g nsp
+  - npm install -g grunt-cli
```

Comment:
Indeed. NPM automatically adds `node_modules/.bin` to the PATH whenever you execute a script, so there's rarely a need to ever globally install anything beyond NPM.